wire-ios/Wire-iOS/Sources/Helpers/syncengine/ZMConversation+Calling.swift

```diff
@@ -116,6 +117,8 @@ extension ZMConversation {
         } else {
             handler(false)
         }
+
+        reachability.tearDown()
```

Comment:
but if you tearDown here, will it not crash when we teardown the environment, maybe we never do...

Reply:
ah ok, environment.reachability is a computed variable so it's not retained by anything...

what about modifying this inside the networkInfo so one doesn't have to remember to do this explicitly? @David-Henner

Reply:
`NetworkInfo` doesn't know anything about `ZMReachability`, it expects an object conforming to the `ServerConnection` protocol. I think the problem has more to do with `ZMReachability` itself. Perhaps as @samwyndham suggested we could do the tear down in the deinit, but there may be a reason it has been done differently